src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Microsoft Azure Management Cognitive Services Library</Description>
-    <VersionPrefix>0.3.0-preview</VersionPrefix>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
@felixwa please replace the entire csproj with the following

<Project Sdk="Microsoft.NET.Sdk">
  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
  <PropertyGroup>
	<PackageId>Microsoft.Azure.Management.CognitiveServices</PackageId>
    <Description>Microsoft Azure Management Cognitive Services Library</Description>
    <VersionPrefix>1.0.0</VersionPrefix>
    <AssemblyName>Microsoft.Azure.Management.CognitiveServices</AssemblyName>
    <PackageTags>Microsoft Azure Cognitive Services management;Cognitive Services;Cognitive Services management;REST HTTP client;windowsazureofficial;netcore451511</PackageTags>
  </PropertyGroup>
  <PropertyGroup>
    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
  </PropertyGroup>
</Project>

If you want to add ReadMe.txt please add .nuspec file, here is an example
https://docs.microsoft.com/en-us/nuget/schema/nuspec#including-content-files